debian/manpages/snapcraft.1

```diff
@@ -0,0 +1,87 @@
+.TH snapcraft 1 "19 July 2016" "version 2.13"
+.SH NAME
+Snapcraft - Snapcraft is a delightful packaging tool
+.SH SYNOPSIS
+snapcraft [options] <command>
+
+snapcraft (-h | --help)
+
+snapcraft --version
```

Comment:
This needs a bunch of delicate formatting.  Something like:

    .B snapcraft
    .RI [ options ]
    .I command
    .br
    .B snapcraft
    .RB [\| \-h \||\| \-\-help \|]
    .br
    .B snapcraft
    .B \-\-version